src/cellrank/estimators/terminal_states/_gpcca.py

```diff
@@ -532,6 +535,140 @@ def set_initial_states(
         )
         return self
 
+    def get_tsi(self, n_macrostates: int, terminal_states: List[str], cluster_key: str, **kwargs: Any) -> pd.DataFrame:
+        """Compute terminal state identificiation (TSI).
+
```

Comment:
Also here, let's later add an explanation/link to the paper for reference.

Reply:
I added a TODO.